app/components/candidate_interface/course_choices_review_visa_status_component.html.erb

```diff
@@ -1,18 +1,20 @@
 <% if can_sponsor_visa? %>
   <%= title %>
 <% else %>
-  <%= govuk_inset_text(classes: 'app-inset-text--narrow-border app-inset-text--important') do %>
+  <%= govuk_inset_text(classes: 'app-inset-text--narrow-border app-inset-text--important govuk-!-margin-top-0 govuk-!-padding-bottom-1') do %>
     <p class="govuk-heading-s app-inset-text__title"><%= title %></p>
 
     <p class="govuk-body">
-      You said that you do not yet have the right to work or study in the UK.
+      Check your eligibility for different <%= govuk_link_to('visa types and immigration statuses', t('train_to_teach_in_england.url')) %>.
     </p>
 
-    <p class="govuk-body">You can:</p>
+    <p class="govuk-body">
+      If you need a Student or Skilled Worker visa to get the right to work or study in the UK, you should find a
+      <%= govuk_link_to('find a course', t('find_postgraduate_teacher_training.production_url')) %> where sponsorship is available.
+    </p>
 
-    <ul class="govuk-list govuk-list--bullet">
-      <li>find a course that can sponsor student visas</li>
-      <li>find out about <%= govuk_link_to('other visas that allow you to study in the UK', t('train_to_teach_in_england.url')) %></li>
-    </ul>
+    <p class="govuk-body">
+      Sponsorship is not available for this course.
```

Comment:
```suggestion
```

Reply:
just realised we say this in the heading of the warning text so don't need to repeat it